engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.controller.js

```diff
@@ -86,8 +87,17 @@ angular.module('Bastion.capsule-content').controller('CapsuleContentController',
                     $scope.syncTask = aggregateTasks(syncStatus['active_sync_tasks']);
 
                 } else if (syncStatus['last_failed_sync_tasks'].length > 0) {
+                    errorCount = $scope.syncTask.humanized.errors.length;
                     $scope.syncTask = pickLastTask(syncStatus['last_failed_sync_tasks']);
-                    $scope.syncErrorMessages = $scope.syncTask.humanized.errors;
+                    if (errorCount > 0) {
+                        $scope.syncErrorMessages = [$scope.syncTask.humanized.errors[0]];
+                    }
+
+                    if (errorCount > 2) {
```

Comment:
Good catch, which means that this will generate two separate alert messages instead of a single string error message which I would think would be preferable.